benchcab/build_cable.py

```diff
@@ -57,7 +58,7 @@ def find_purge_line(filelines, filename=""):
     return purge_line
 
 
-def change_build_lines(filelines, modules, filename=""):
+def change_build_lines(filelines, modules, filename=Path):
```

Comment:
As per the comment above
```suggestion
def change_build_lines(filelines, modules, filename):
```